packages/web/src/apollo/index.tsx

```diff
@@ -188,12 +195,26 @@ const ApolloProviderWithFetchConfig: React.FunctionComponent<{
     httpLink = new HttpLink({ uri, fetch: crossFetch, ...httpLinkConfig })
   }
 
-  // The order here is important. The last link *must* be a terminating link like HttpLink.
+  // Our terminating link needs to be smart enough to handle subscriptions, and if the GraphQL query
+  // is subscription it needs to use the SSELink (server sent events link).
+  const terminatingLink = new RetryLink().split(
+    ({ query }) => {
+      const definition = getMainDefinition(query)
+      return (
+        definition.kind === 'OperationDefinition' &&
+        definition.operation === 'subscription'
+      )
+    },
+    new SSELink({ url: uri, headers }),
```

Comment:
@dthyresson I'm wondering if the `headers` here are incomplete—this is relying on the headers returned by the `useFetchConfig` hook. The headers are enriched by the `authMiddleware` function. Maybe we need to use the apollo link context to get the headers here?

Reply:
Ah yes I wondered about that too. I need to test that authentication still applies in bit classic queries and also the subscription sse link.

Reply:
Auth now works! Woo hoo. TLDR; the auth token logic needed to be replicated in the SSELink constructor.

The SSE client now also maps other headers for referrer, credentials (ie, CORS), and referrerPolicy